src/Microsoft.Health.Fhir.Core/Features/Conformance/SystemConformanceProvider.cs

```diff
@@ -98,6 +104,38 @@ public override async Task<ResourceElement> GetCapabilityStatementOnStartup(Canc
             return _listedCapabilityStatement;
         }
 
+        public async void RebuildCapabilityStatement()
```

Comment:
We should avoid having `async void` methods

Reply:
Fixed